.github/workflows/e2e_miscellaneous_tests.yml

```diff
@@ -66,6 +66,11 @@ jobs:
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite="contributorDashboard" --prod_env
         env:
           VIDEO_RECORDING_IS_ENABLED: 0
+      - name: Run e2e Contributor Dashboard Tests in mobile view
+        if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
+        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-install --skip-build --suite="contributorDashboard" --mobile --prod_env
+        env:
+          VIDEO_RECORDING_IS_ENABLED: 0
```

Comment:
Can you manually check that video recording still works with the mobile view?

Reply:
I have pushed a commit with this enabled. Will check if the recording worked.

Reply:
I am unable to trigger the video recording consistently on github actions even after turning the flag on. (This is the case with other e2e tests as well)

Reply:
This is a problem. Video recordings are being used to debug e2e tests right now (e.g. https://github.com/oppia/oppia/actions/runs/3052491216), so they appear to be working on develop. This suggests that something in this PR broke the video recordings. Can you investigate?

Reply:
Ah it seems like this issue was happening due to me turning on the wrong flag (my bad). The video recording seems to work as expected. Here is the run: [https://github.com/oppia/oppia/actions/runs/3077901960](https://github.com/oppia/oppia/actions/runs/3077901960)